app/assets/javascripts/modules/external_users/claims/fee_calculator/FeeCalculator.UnitPrice.js

```diff
@@ -52,19 +53,23 @@
     // It should be a part of the checkbox logic and located in a module
     // related to that.
     markForDestruction: function (context, bool) {
+      console.log(context);
+      console.log(bool);
```

Comment:
A couple of JS debug statements that can go.

Reply:
👍 removed in later commit